dlp/src/main/java/com/example/dlp/Inspect.java

```diff
@@ -332,9 +334,79 @@ private static void inspectDatastore(
     // [END dlp_inspect_datastore]
   }
 
+  private static void inspectBigquery(
+          String projectId,
+          String datasetId,
+          String tableId,
+          Likelihood minLikelihood,
+          List<InfoType> infoTypes) {
+    // [START dlp_inspect_bigquery]
+    // Instantiates a client
+    try (DlpServiceClient dlpServiceClient = DlpServiceClient.create()) {
+
+      // (Optional) The project ID to run the API call under
```

Comment:
You might wish to mention the API's if you are going to mention this much stuff.

Reply:
Perhaps in a more JavaDoc way and move the tag earlier.

Reply:
Done.